.github/workflows/dogfood.yml

```diff
@@ -0,0 +1,55 @@
+name: dogfood merge
+# This workflow is triggered on pushes to master and dogfood branches
+on:
```

Comment:
Educational: I guess this `on` cannot be completely removed, without making the `yml` invalid, right?

Reply:
Why would you want to remove it?

Reply:
Nevermind, you still need to tell on which actions. All good, I misunderstood the section.